test/views/declaration/spec/UnitViewSpec.scala

```diff
@@ -39,6 +42,27 @@ class UnitViewSpec extends UnitSpec with ViewMatchers {
       "error.summary.text"
     )
   }
+
+  def messagesKey(key: String): BeMatcher[String] = new MessagesKeyMatcher(key)
+}
+
+class MessagesKeyMatcher(key: String) extends BeMatcher[String] {
```

Comment:
Is this used anywhere in this PR?  Can't see it.

Reply:
Probably github hide diff on view spec test/views/declaration/DocumentsProducedViewSpec.scala - whole view spec have been rewritten by me and it was used there